source/.vscode/settings.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
This file should not be on the github. Please remove it from the source and eventually add an ignore entry if needed

Reply:
ok thank you